src/_includes/how_to/get_connected_plan_table.html

```diff
@@ -2,23 +2,37 @@
   <tr>
     <th scope="col" class="border-top align-middle table-dark-blue border-dark-blue text-white table-bordered">1 GB</th>
     <td class="border-top border-dark-blue table-bordered">
-      <li class="mx-4">GTFS-RT <em>or</em> 1 fare payment validator</li>
+      <ul class="p-0 m-0">
+        <li class="mx-4">GTFS-RT <em>or</em> 1 fare payment validator</li>
+      </ul>
     </td>
   </tr>
   <tr>
     <th scope="col" class="align-middle table-dark-blue border-dark-blue text-white table-bordered">2 GB</th>
     <td class="border-dark-blue table-bordered">
-      <li class="mx-4">GTFS-RT</li>
-      <li class="mx-4">2 fare payment validators</li>
+      <ul class="p-0 m-0">
+        <li class="mx-4">GTFS-RT</li>
+      </ul>
+      <ul class="p-0 m-0">
+        <li class="mx-4">2 fare payment validators</li>
+      </ul>
     </td>
   </tr>
   <tr>
     <th scope="col" class="align-middle table-dark-blue border-dark-blue text-white table-bordered">Unlimited</th>
     <td class="border-dark-blue table-bordered">
-      <li class="mx-4">GTFS-RT</li>
-      <li class="mx-4">2 fare payment validators</li>
-      <li class="mx-4">Cameras</li>
-      <li class="mx-4">CAD/AVL</li>
+      <ul class="p-0 m-0">
```

Comment:
Can this styling be achieved with a single, semantically-correct `<ul`> wrapping all these `<li>`? It's strange to have many single-item `<ul>` next to each other, when there is really just one list of items.

Reply:
https://github.com/cal-itp/mobility-marketplace/pull/465/commits/a657f79c2d6f610951edd8f2e2130b74642b4bbe